source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -110,442 +110,528 @@ volume and a supporting
+         Modify the generated YAML file as desired, then use ``kubectl apply -f <FILE>`` to manually create the MinIO tenant using the file.
+
+      4) Record the Access Information
+
+         When generating the tenant, the MinIO Operator displays the access credentials to use for the tenant.
+
+         .. important::
+   
+            This is the only time the credentials display.
+            They cannot be retrieved later.
+
+            Record the username and password to a secure location before closing the terminal.
+
+         In addition to access credentials, the output shows the service name and service reports to use for accessing the tenant.
+
+      5) Access the Tenant's MinIO Console
+
+         To access the :ref:`MinIO Console <minio-console>` for the tenant, forward the tenant's port.
+
+         - If necessary, run ``kubectl get svc -n <namespace>`` to retrieve the tenant's port number.
+  
+         - Run the following to forward the tenant's port and access it from a browser:
+
+            .. code-block:: shell
+               :class: copyable
+
+               kubectl port-forward svc/<tenant-name>-console -n <tenant-namespace> <localport>:<tenantport>
+
+           - Replace ``<tenant-name>`` with the name of your tenant.
+  
+           - Replace ``<tenant-namespace>`` with the namespace the tenant exists in.
+  
+           - Replace ``<localport>`` with the port number to use on your local machine to access the tenant's MinIO Console.
+  
+           - Replace ``<tenantport>`` with the port number the MinIO Operator assigned to the tenant.
+
+         - Go to ``https://127.0.0.1:<localport>`` to Access the tenant's MinIO Console.
  
-By default each service is visible only within the Kubernetes cluster. 
-Applications deployed inside the cluster can access the services using the 
-``CLUSTER-IP``. 
-
-Applications external to the Kubernetes cluster can access the services using
-the ``EXTERNAL-IP``. This value is only populated for Kubernetes clusters 
-configured for Ingress or a similar network access service. Kubernetes provides
-multiple options for configuring external access to services. See the Kubernetes
-documentation on :kube-docs:`Publishing Services (ServiceTypes)
-<concepts/services-networking/service/#publishing-services-service-types>` and
-:kube-docs:`Ingress <concepts/services-networking/ingress/>` for more complete
-information on configuring external access to services.
-
-You can temporarily expose each service using the 
-``kubectl port-forward`` utility. Run the following examples to forward 
-traffic from the local host running ``kubectl`` to the services running inside 
-the Kubernetes cluster.
+           Replace ``<localport>`` with the port number you used when forwarding the tenant's port.
+
+         - Login with the username and password shown in the tenant creation output and recorded in step 4 above.
+
+Forward Ports
```

Comment:
This feels oddly alone - there's no context on where to use this.

You can just take the instructions and add it to the "Connect" step for each procedure.

Reply:
Moved to be an additional step on each procedure.